<ui version="4.0" >
 <class>Disconnect</class>
 <widget class="QDialog" name="Disconnect" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Sinerji</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff>images/icon.png</normaloff>images/icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" >
       <item>
        <widget class="QLabel" name="iconDisLabel" >
         <property name="text" >
          <string/>
         </property>
         <property name="pixmap" >
          <pixmap>images/connect.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="clientTextLabel" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <spacer name="horizontalSpacer" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>140</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="disconnectButton" >
         <property name="text" >
          <string>Disconnect</string>
         </property>
         <property name="icon" >
          <iconset>
           <normaloff>images/disconnect.png</normaloff>images/disconnect.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="okButton" >
         <property name="text" >
          <string>Ok</string>
         </property>
         <property name="icon" >
          <iconset>
           <normaloff>images/buttonApply.png</normaloff>images/buttonApply.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
